libpius/signer.py

```diff
@@ -33,10 +33,14 @@ class PiusSigner(object):
   GPG_SIG_BEG = '[GNUPG:] BEGIN_SIGNING'
   GPG_SIG_CREATED = '[GNUPG:] SIG_CREATED'
 
-  def __init__(self, signer, mode, keyring, gpg_path, tmpdir, outdir,
-               encrypt_outfiles, mail, mailer, verbose, sort_keyring,
+  def __init__(self, signer, force_signer, mode, keyring, gpg_path, tmpdir,
+               outdir, encrypt_outfiles, mail, mailer, verbose, sort_keyring,
                policy_url, mail_host):
     self.signer = signer
+    self.force_signer = force_signer
+    self.forced_signer = self.signer
```

Comment:
Yes exactly. In my case the main key has the signing capability and I think all GnuPG keys have that (to sign the subkeys) but maybe someone wants to use another specific subkey for the signing.